cpp/include/types.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef TYPES_HPP
+#define TYPES_HPP
+
+#include <algorithm>
+#include <cassert>
+#include "cudf.h"
+
+namespace cudf {
+
+/**
+ * @brief A wrapper for a set of gdf_columns all with equal number of rows.
+ *
+ */
+struct table {
```

Comment:
Why not call this a DataFrame? It's a DataFrame library.

Reply:
See my response to your other comment.